spec/index.html

```diff
@@ -5462,29 +5462,53 @@ <h5>RDFterm-equal</h5>
                                                                                    "type">RDF term</span> <span class="name">term1</span> <span class=
                                                                                                                                                 "operator">=</span> <span class="type">RDF term</span> <span class=
                                                                                                                                                                                                              "name">term2</span></pre>
-            <p>Returns TRUE if <code>term1</code> and <code>term2</code> are the same RDF term as
-              defined in [[[RDF12-CONCEPTS]]] [[RDF12-CONCEPTS]]; produces a type error if the
-              arguments are both literal but are not the same RDF term <sup><a href=
-                                                                               "#func-RDFterm-equal-foot1" class="footnote">*</a></sup>; returns FALSE otherwise.
-              <code>term1</code> and <code>term2</code> are the same if any of the following is
+            <p>The behavior of this operator is defined as follows:</p>
+            <ul>
+                <li>Returns TRUE if <code>term1</code> and <code>term2</code> are <em>equal</em> RDF terms, as defined below.</li>
+                <li>Produces a type error if <code>term1</code> and <code>term2</code> are both literals that have the
+              same <a data-cite="RDF12-CONCEPTS#dfn-datatype-iri">datatype IRI</a>, this datatype IRI is <em>not</em> in the
+              set of <a data-cite="RDF12-CONCEPTS#dfn-recognized-datatype-iri">recognized datatype IRIs</a>.</li>
```

Comment:
Notice that these two points are not mutually exclusive and their order matters. I would try to avoid that and make sure that every case fits only one of these points (which would then make the order of these points irrelevant).

The issue is with cases in which the two terms are literals that have i) the same datatype IRI, ii) this IRI is not one of the recognized datatype IRIs, and iii) the two literals have the same lexical form (i.e., they are actually the same literal). This case is covered by the first point, but also by the second one. By the first point, the function returns TRUE in this case, and I think that this is the expected behavior.

Now, to make sure that this case does not also fit the second point (and, thus, to make the order of the points irrelevant) I propose to rephrase the second point as follows.
```suggestion
                <li>Returns TRUE if <code>term1</code> and <code>term2</code> are <em>equal</em> RDF terms, as defined below.</li>
                <li>Produces a type error if <code>term1</code> and <code>term2</code> are both literals such that these two literals have the
              same <a data-cite="RDF12-CONCEPTS#dfn-datatype-iri">datatype IRI</a>, this datatype IRI is <em>not</em> in the
              set of <a data-cite="RDF12-CONCEPTS#dfn-recognized-datatype-iri">recognized datatype IRIs</a>, and the
              two literals have the same <a data-cite="RDF12-CONCEPTS#dfn-lexical-form">lexical form</a>.</li>
```

Reply:
I agree, making the different cases mutually exclusive would make things clearer.

However, instead of

> two literals have the same <a data-cite="RDF12-CONCEPTS#dfn-lexical-form">lexical form</a>.

in the second case, I suspect you mean

> two literals a different <a data-cite="RDF12-CONCEPTS#dfn-lexical-form">lexical form</a>.

?

Reply:
Yes, that's what I meant ;-)

Here is the updated suggestion, ready for you to commit.
```suggestion
                <li>Returns TRUE if <code>term1</code> and <code>term2</code> are <em>equal</em> RDF terms, as defined below.</li>
                <li>Produces a type error if <code>term1</code> and <code>term2</code> are both literals such that these two literals have the
              same <a data-cite="RDF12-CONCEPTS#dfn-datatype-iri">datatype IRI</a>, this datatype IRI is <em>not</em> in the
              set of <a data-cite="RDF12-CONCEPTS#dfn-recognized-datatype-iri">recognized datatype IRIs</a>, and the
              <a data-cite="RDF12-CONCEPTS#dfn-lexical-form">lexical forms</a> of the two literals are different from one another.</li>
```